site/src/Librecores/ProjectRepoBundle/Entity/Project.php

```diff
@@ -71,6 +72,15 @@ class Project
      */
     private $parentOrganization;
 
+    // Associations
+    /**
+     * @var ArrayCollection
+     *
+     * @ORM\OneToMany(targetEntity="ProjectClassification", mappedBy="project")
+     * @ORM\JoinColumn(name="projectId", referencedColumnName="id")
+     */
+    protected $projects;
```

Comment:
This item is named a bit misleadingly. Don't associate a project with another project, associate a project with a ProjectClassification entity. So "protected $classifications" would work.